tools/whisper/whisper.xml

```diff
@@ -0,0 +1,338 @@
+            <option value="Nynorsk">Nynorsk</option>
+            <option value="Occitan">Occitan</option>
+            <option value="Panjabi">Panjabi</option>
+            <option value="Pashto">Pashto</option>
+            <option value="Persian">Persian</option>
+            <option value="Polish">Polish</option>
+            <option value="Portuguese">Portuguese</option>
+            <option value="Punjabi">Punjabi</option>
+            <option value="Pushto">Pushto</option>
+            <option value="Romanian">Romanian</option>
+            <option value="Russian">Russian</option>
+            <option value="Sanskrit">Sanskrit</option>
+            <option value="Serbian">Serbian</option>
+            <option value="Shona">Shona</option>
+            <option value="Sindhi">Sindhi</option>
+            <option value="Sinhala">Sinhala</option>
+            <option value="Sinhalese">Sinhalese</option>
+            <option value="Slovak">Slovak</option>
+            <option value="Slovenian">Slovenian</option>
+            <option value="Somali">Somali</option>
+            <option value="Spanish">Spanish</option>
+            <option value="Sundanese">Sundanese</option>
+            <option value="Swahili">Swahili</option>
+            <option value="Swedish">Swedish</option>
+            <option value="Tagalog">Tagalog</option>
+            <option value="Tajik">Tajik</option>
+            <option value="Tamil">Tamil</option>
+            <option value="Tatar">Tatar</option>
+            <option value="Telugu">Telugu</option>
+            <option value="Thai">Thai</option>
+            <option value="Tibetan">Tibetan</option>
+            <option value="Turkish">Turkish</option>
+            <option value="Turkmen">Turkmen</option>
+            <option value="Ukrainian">Ukrainian</option>
+            <option value="Urdu">Urdu</option>
+            <option value="Uzbek">Uzbek</option>
+            <option value="Valencian">Valencian</option>
+            <option value="Vietnamese">Vietnamese</option>
+            <option value="Welsh">Welsh</option>
+            <option value="Yiddish">Yiddish</option>
+            <option value="Yoruba">Yoruba</option>
+        </param>
+        <param argument="--output_format" type="select" multiple="true" label="Output Format">
+            <option value="txt" selected="true">Text</option>
+            <option value="json">JSON</option>
+            <option value="srt">SubRip</option>
+            <option value="vtt">WebVTT</option>
+            <option value="tsv">Tab-separated values</option>
+        </param>
+        <section name="advanced" title="Advanced Options">
+            <param name="temperature" type="integer" value="0" optional="true" label="Temperature" help="Temperature to use for sampling" />
+            <param name="best_of" type="integer" value="5" optional="true" label="Best of" help="Number of candidates when sampling with non-zero temperature" />
```

Comment:
Best of what? the label is not very intuitive

Reply:
The help attribute explains it. Is it not enough? How about replacing it with help text?

Reply:
look at the interface. I think its is confusing and not helpful. You read the short description ... that is the label and it should be meaningful. If you need advanced information you read the help.